tcms/testcases/models.py

```diff
@@ -346,13 +353,8 @@ def get_components(self):
     def get_component_names(self):
         return self.component.values_list('name', flat=True)
 
-    def get_choiced(self, obj_value, choices):
-        for x in choices:
-            if x[0] == obj_value:
-                return x[1]
-
     def get_is_automated(self):
-        return self.get_choiced(self.is_automated, AUTOMATED_CHOICES)
+        return _get_choiced(self.is_automated, AUTOMATED_CHOICES)
```

Comment:
`_get_choiced()` is used only here, so it doesn't make sense to have it as a separate function. In addition `get_is_automated()` is used only in `get_is_automated_status()` so no need to have it as a separate method.